mcs/class/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -3,6 +3,15 @@
   <!-- Include common build properties -->
 <!--  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" /> -->
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <!-- Default values -->
+  <PropertyGroup Condition="'$(TargetsUnix)'=='' AND '$(TargetsOSX)'=='' AND '$(TargetsWindows)'==''">
+    <TargetsUnix>true</TargetsUnix>
+    <TargetsOSX>true</TargetsOSX>
+    <TargetsWindows>false</TargetsWindows>
+    <BuildArch>x64</BuildArch>
+    <OutputPath>bin/x64</OutputPath>
+    <FeaturePortableTimer>true</FeaturePortableTimer>
```

Comment:
nit: This one can be defined unconditionally and removed from the Makefile.

Reply:
I'd leave makefile as is - as an example how to configure build. These default values are for OSX-64 only (also as an example)

Reply:
I meant specifically the `<FeaturePortableTimer>true</FeaturePortableTimer>` line. It's used across all platforms. Both CoreRT and CoreCLR specify the `Feature*` definitions inside the .csproj and optionally condition them based on `Targets*` definition.

Reply:
ah, ok, makes sense then